resources/dark/conclusions/skip.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Rename this file to "skipped.svg" so that it matches the conclusion response property.

See [here](https://developer.github.com/v3/checks/runs/#parameters)

Reply:
Nice catch!